spec/functional/collections.spec.js

```diff
@@ -70,7 +70,7 @@ describe('collections', function () {
             first: 'https://localhost/followers/test?page=true'
           }
           expect(res.body).toEqual(standard)
-          done(err)
+          done()
```

Comment:
Correct me if I'm wrong, but I think this could allow tests to pass even if an unexpected error is returned from supertest. I think we would need to `if (err) { throw err }` to be sure we didn't have false negatives (admittedly the  expect statements would probably still cause the test to fail in most cases, but not in a "exepct y not to have changed" type test, so I'd rather avoid the pattern)

Reply:
Thanks for the review. Yes, I think you're correct. I'll go back and review those changes. I haven't used Jasmine before, but I dig some more digging in the documentation and it appears that `done.fail(err)` is the replacement for `done(err)`, so I can use that idiom as well.